cmd/cmd.go

```diff
@@ -132,13 +156,23 @@ func directoryHandler(dirname string) (rftp.FileHandler, error) {
 }
 
 func init() {
-	rootCmd.PersistentFlags().BoolVarP(&s, "server", "s", false, `server mode: accept incoming files from any host
-Operate in client mode if “–s” is not specified`)
+	rootCmd.PersistentFlags().BoolVarP(&s, "server", "s", false, `server mode:
+accept incoming files from any host Operate in client mode if “–s” is not
+specified`)
+
 	rootCmd.PersistentFlags().IntVarP(&t, "port", "t", 0, "specify the port number to use")
-	rootCmd.PersistentFlags().Float32VarP(&p, "p", "p", -1, `specify the  loss probabilities for the Markov chain model (0 <= p <= 1)
-if only one is specified, assume p=q; if neither is specified assume no loss`)
-	rootCmd.PersistentFlags().Float32VarP(&q, "q", "q", -1, `specify the  loss probabilities for the Markov chain model (0 <= p <= 1)
-if only one is specified, assume p=q; if neither is specified assume no loss`)
+
+	rootCmd.PersistentFlags().Float32VarP(&p, "p", "p", -1, `specify the loss
+probabilities for the Markov chain model (0 <= p <= 1) if only one is specified,
+assume p=q; if neither is specified assume no loss`)
+
+	rootCmd.PersistentFlags().Float32VarP(&q, "q", "q", -1, `specify the loss
+probabilities for the Markov chain model (0 <= p <= 1) if only one is specified,
+assume p=q; if neither is specified assume no loss`)
+
+	rootCmd.PersistentFlags().StringVarP(&out, "out", "o", ".", `specify the
```

Comment:
I think I understand the line breaks for most of the flags and I think they make sense, but the breaks in the `out` flag look wrong to me:
![2020-07-22-194415_grim](https://user-images.githubusercontent.com/8409346/88210026-d40a0600-cc53-11ea-8963-df421d5f8fc0.png)